bin/deployment.rb

```diff
@@ -12,4 +12,12 @@ def deployments
       end
     end
   end
+
+  def exclude_list
+    @exclude_list ||= [].tap do |exclude_list|
```

Comment:
We don't need to memoize this method because we are only using it in one place. This is enough:
```ruby
[].tap do |exclude_list|
```

Reply:
@arturoherrero we using it in one place but multiple time:
```
      Dir[File.join("config", "deploy", "*.rb")].each do |file|
        deployment = file.split("/").last[0..-4]
        next if deployment == "deployment" || exclude_list.include?(deployment)
        puts "Include #{deployment}? (Y/n): "
```
It will be called for each file in that folder. 
Although I can change the logic and assign the value of `exclude_list` to a local variable before the loop inside `deployments`, but with current implementation, it will be called multiple time and memorizing it will increase performance.

Reply:
You are right 👍